CHANGELOG.md

```diff
@@ -1,3 +1,513 @@
+## [Talos 0.11.1](https://github.com/talos-systems/talos/releases/tag/v0.11.1) (2021-07-14)
+
+Welcome to the v0.11.1 release of Talos!
+
+
+
+Please try out the release binaries and report any issues at
+https://github.com/talos-systems/talos/issues.
+
+### Default to Bootstrap workflow
+
+The `init.yaml` is no longer an output of `talosctl gen config`.
+We now encourage using the bootstrap API, instead of `init` node types, as we
+intend on deprecating this machine type in the future.
+The `init.yaml` and `controlplane.yaml` machine configs are identical with the
+exception of the machine type.
+Users can use a modified `controlplane.yaml` with the machine type set to
+`init` if they would like to avoid using the bootstrap API.
+
+
+### Component Updates
+
+* containerd was updated to 1.5.2
+* Linux kernel was updated to 5.10.45
+* Kubernetes was updated to 1.21.2
+* etcd was updated to 3.4.16
+* CoreDNS was updated to 1.8.4
+
+
+### CoreDNS
+
+Added the flag `cluster.coreDNS.disabled` to coreDNS deployment during the cluster bootstrap.
```

Comment:
Found my typo here :facepalm: 
```suggestion
Added the flag `cluster.coreDNS.disabled` to disable coreDNS deployment during the cluster bootstrap.
```

Reply:
I'll get it fixed, but might have slipped already into the docs

Reply:
Into releases I guess. And maybe we can edit them manually?